frameworks/Python/bottle/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-export PY2_ROOT=$IROOT/py2
-export PY2=$PY2_ROOT/bin/python
-export PY2_PIP=$PY2_ROOT/bin/pip
-export PY2_GUNICORN=$PY2_ROOT/bin/gunicorn
 
-$PY2_GUNICORN app:app -c gunicorn_conf.py &
\ No newline at end of file
+fw_depends python2
+
+$PY2_ROOT/bin/pip install --install-option="--prefix=${PY2_ROOT}" -r $TROOT/requirements.txt
+
+gunicorn app:app -c gunicorn_conf.py &
```

Comment:
Not necessary anymore. Unfortunately, the line about it still requires `$PY2_ROOT/bin/pip` because our software suite is written in Python and requires `pip` prior to running, so there is a name collision on the path. However, `gunicorn` is NOT on the path prior to running a benchmark, and `fw_depends python2` tell it to put `$PY2_ROOT/bin` onto the path, so `gunicorn` is sufficient.